public/javascripts/timeline.js

```diff
@@ -43,35 +50,115 @@ function update () {
     this.now_offset = .25 * hour_tick_list.offsetWidth + ((now - selectedDate) / MS_PER_HOUR ) * this.hour_width;
 
 
-    // Calculate the start point of the first hour
-    var hours_to_left = this.now_offset / this.hour_width;
+    // hours being displayed
+    if (slider_value > 3 && slider_value <= 8) {
```

Comment:
I'm a bit uncomfortable basing this change off of the slider value. The algorithm that determines how the slider value maps to the duration displayed could be changed as the site evolves. I'd rather not couple the slider value to the minutes-display directly.

A better option may be hour_width which is the pixel width of a displayed hour. Since this is what the user will see, and the display becomes useless if hour_width becomes too high, this would be a good metric for transferring to viewing minutes.